### PR TITLE
NO_DIST in daal4py CI

### PR DESCRIPTION
**Pull request by OnlyDeniko**

### CODE REVIEW

File: .ci/pipeline/ci.yml [hunk 1]
```diff
@@ -189,6 +190,7 @@ jobs:
       source ${DALROOT}/env/vars.sh
       export TBBROOT=/opt/intel/oneapi/tbb/latest
       source ${TBBROOT}/env/vars.sh
+      export NO_DIST=1
```
Comment (PetrovKP) [inline .ci/pipeline/ci.yml:193]: Why it is need?

**Comment by OnlyDeniko** [comment]
Problem with DF sycl examples was before my PR, so this PR is not affect on it

**Review by PetrovKP — APPROVED** [review]

**Comment by PetrovKP** [comment]
I think it can merge. Problem with DF will fixed in other PR

File: .ci/pipeline/ci.yml (hunk 1 above)
Comment (OnlyDeniko) [reply .ci/pipeline/ci.yml]: removed

**Pull request merged**